internal/notification/notifier.go

```diff
@@ -2,5 +2,5 @@ package notification
 
 // Notifier interface for sending the identity verification link.
 type Notifier interface {
-	Send(to string, subject string, link string) error
+	Send(recipient string, subject string, body string) error
```

Comment:
You can use a short declaration for the `string` types here.

Reply:
Is there any benefit to it? I figured them matching in the interface would reduce any confusion. My experience with go is limited. Or do you mean "recipient, subject string" due to the duplicate type?

Reply:
Yes I mean because all of the types are string you don't need to declare explicitly for each, so:
`Send(recipient, subject, body string) error`.

@clems4ever this is probably something we should take care of in in linter/styler service perhaps.

Reply:
I can do that for this PR. I only didn't do it because the code was readable as-is at native 1080p. Let me know if you'd like me to though.

Reply:
Done, also did file_notifier.go